debian/control

```diff
@@ -14,19 +14,18 @@ Build-Depends:
  , debhelper (>= 9)
  , dh-autoreconf
 # C/C++
- , g++ (>= 4:4.7)
```

Comment:
wondering why this dependency is not needed

Reply:
My hunch is because `g++` is part of `build-essential` (which, IIRC, is always pulled in pbuilder)